issue_03/script.js

```diff
@@ -0,0 +1,16 @@
+const ul = document.getElementById('js-ul');
+const hrefArray = ['a1.html', 'a2.html'];
+const textArray = ['a1', 'a2'];
+const srcArray = ['/img/bookmark.png', '/img/message.png'];
+
+for (let i = 0; i < 2; i++) {
+    const li = document.createElement('li');
+    const a = document.createElement('a');
+    const img = document.createElement('img');
+
+    a.setAttribute('href', hrefArray[i]);
+    a.textContent = textArray[i];
+    img.setAttribute('src', srcArray[i]);
+    a.insertAdjacentElement('afterbegin', img);
+    ul.appendChild(li).appendChild(a);
```

Comment:
IMO
You can even write it like this!
```suggestion
ul.appendChild(li).appendChild(a).insertAdjacentElement('afterbegin', img);
```

Reply:
I must have missed that!
Thanks for pointing it out :+1: